library/L1_Peripheral/lpc40xx/timer.hpp

```diff
@@ -130,6 +135,26 @@ class Timer final : public sjsu::Timer
     return timer_.peripheral->TC;
   }
 
+  void Start() const override
+  {
+    timer_.peripheral->TCR = bit::Set(
+        timer_.peripheral->TCR, TimerControlRegister::kEnableBit.position);
```

Comment:
```suggestion
        timer_.peripheral->TCR, TimerControlRegister::kEnableBit);
```
I added a version of this that does not require you to pull out the position field.